### ISSUE TITLE
Update models documentation with current types

### ISSUE THREAD

**Issue by danielhers** [explosion/spaCy#1513]
In the [models documentation](https://spacy.io/models/#available-models), the model types are different than the latest [release notes](https://github.com/explosion/spaCy/releases/tag/v2.0.0). For example, `en_core_web_sm` has "vectors" there even though it is no longer true.

And a related question - is there a German model with word vectors? I only see `de_core_news_sm` in the release not. I believe v1 did have German word vectors...

**Comment by ines** [comment]
Ah yes, I agree – the way this is displayed in the table there is a bit unideal. It should probably look more like the [one in the release notes](https://github.com/explosion/spaCy/releases/tag/v2.0.0) and not list the vectors for the `sm` models. Will fix this!

We've tried to be more explicit about the context-sensitive token vectors vs. word vectors in the usage docs and the models overview:
* https://spacy.io/usage/vectors-similarity (see first section)
* https://spacy.io/models/en (see "context vectors only" vs. the word vector details)

> And a related question - is there a German model with word vectors? I only see `de_core_news_sm` in the release not. I believe v1 did have German word vectors...

`de_core_news_sm` only has context vectors, yes – so you can use the `similarity()` methods, but not `.vector` etc. There will also be an `de_core_news_md` model with word vectors (similar to the English and Spanish one) – it did finish training last night, but fell over during packaging 😓 I think we should be able to recover it and won't have to retrain, so fingers crossed!

**Comment by danielhers** [comment]
Excellent, thanks!

**Comment by danielhers** [comment]
The description page for `en_core_web_sm` still shows "vectors": https://spacy.io/models/en#en_core_web_sm

**Comment by ines** [comment]
@danielhers Yes, but it shows up as "context vectors only", right? I wrote a little more about this in #1520 – this all comes down to the problem of getting across that the `sm` models do in fact contain some form of vectors that let you use the `similarity()` methods, but not actually word vectors.

**Comment by danielhers** [comment]
I see, thanks. Perhaps it would be clearer to write either "word vectors" or "context vectors" instead of "vectors".

**Comment by danielhers** [comment]
@ines how is `de_core_news_md` doing, if I may ask?
